modules/perc-distribution-tree/src/main/resources/distribution/rxconfig/Installer/data/cmsTableDef.xml

```diff
@@ -361,7 +361,7 @@
                 <size>500</size>
                 <allowsnull>yes</allowsnull>
             </column>
-            <column name="TRANSITIONCOMMENT" >
+            <column action="r" name="TRANSITIONCOMMENT" >
```

Comment:
What does "r" do?  Can you add a comment in the XML file here.

Reply:
'r' means Replace

Reply:
Added comment